.github/workflows/build.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         with:
           args: 
              "${{ steps.imagemagick.outputs.version }}"
+      - name: Test 
+        run:  rpm -qp --requires rpms/ImageMagick-libs-${{ steps.imagemagick.outputs.version }}.x86_64.rpm 
```

Comment:
Added "egrep -i " which basically trying to find the dependencies which we have mentioned. egrep does found any mentioned dependencies hence existing with error. So to continue the workflow added "continue-on-error"

(checked locally grep -v again listing the complete list of dependencies )

Reply:
we can't use `continue-on-error` or the build status will be unaffected if grep is matching or not.

you should try with `-L` that is inverse match for the entire file instead of the single line as with `-v`

```
bash-3.2$ cat test.txt
abc
def
ghi
bash-3.2$ ! cat test.txt | grep -L def
bash-3.2$ echo $?
1
bash-3.2$ ! cat test.txt | grep -qL zzz
bash-3.2$ echo $?
0
```

notice that I had to use `!` to negate the whole command since we want to fail when it match the pattern and success otherwise